src/test/java/io/supertokens/test/ThirdPartyUsersAPITest2_7.java

```diff
@@ -120,4 +120,113 @@ public void testBadInput() throws Exception {
         process.kill();
         assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STOPPED));
     }
+
+    //-- Check good input works (add 5 users)
+    // *    - no params passed should return 5 users
+    // *    - only limit passed (limit: 2. users are returned in ASC order based on timeJoined)
+    // *    - limit and timeJoinedOrder passed (limit: 2, timeJoinedOrder: DESC. users are returned in DESC order
+    // based on timeJoined)
+    @Test
+    public void testGoodInput() throws Exception {
+        String[] args = {"../"};
+
+        TestingProcessManager.TestingProcess process = TestingProcessManager.start(args);
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STARTED));
+
+        ThirdParty.signInUp(process.getProcess(), "thirdPartyId",
+                "thirdPartyUserId", "test@example.com", false);
+        Thread.sleep(100);
```

Comment:
Write a comment for why you have thread.sleep(100)

Reply:
added comment to explain the addition of Thread.sleep